roles/qe_common/README.md

```diff
@@ -0,0 +1,58 @@
+qe_common
+=========
+
+The tests in this role are not specific to any one functional area.
+
+Requirements
+------------
+
+None
+
+Role Variables
+--------------
+
+This role should not have any default variables as this is a common role for tests that can be used for multiple test roles.
```

Comment:
This section should define the available variables that are used by this role, rather than describing the default vars.

Reply:
fixed